web/src/tippyjs.ts

```diff
@@ -7,11 +7,15 @@ import render_change_visibility_policy_button_tooltip from "../templates/change_
 import render_org_logo_tooltip from "../templates/org_logo_tooltip.hbs";
 import render_tooltip_templates from "../templates/tooltip_templates.hbs";
 
+import * as drafts from "./drafts.ts";
 import {$t} from "./i18n.ts";
 import * as people from "./people.ts";
+import * as scheduled_messages from "./scheduled_messages.ts";
 import * as settings_config from "./settings_config.ts";
+import * as starred_messages from "./starred_messages.ts";
 import * as stream_data from "./stream_data.ts";
 import * as ui_util from "./ui_util.ts";
+import * as unread from "./unread.ts";
```

Comment:
We probably should aim to limit how many feature-specific modules this one imports to avoid creating import cycles and maintain good abstractions. 

Probably in practice what that means is we should make a new `left_sidebar_tooltips.ts` that declares the left-sidebar tooltips and imports `tippyjs.ts` for anything it needs. `web/src/compose_tooltips.ts` and `web/src/message_list_tooltips.ts` are examples.

Can be a quick follow-up PR if I'm able to merge this now.